src/Mod/Draft/importDWG.py

```diff
@@ -302,6 +302,8 @@ def convertToDwg(dxffilename, dwgfilename):
             proc.communicate()
             return dwgfilename
         except Exception:
-            FCC.PrintError(translate("draft", "QCAD error") + "\n")
-
+            if conv != 0:
+                FCC.PrintError(translate("draft", "QCAD error") + "\n")
+            else:
+                FCC.PrintError(translate("draft", "No suitable DWG convertor has been found. Please set one manually under menu Edit -> Preferences -> Import/Export -> DWG") + "\n")
```

Comment:
Since the message is quite long, I would add a linebreak after the first sentence ends
Also Lint reports the code line as such is too long (we have a 100 char limit)